docs/developer/release/maven-artifacts/dataflow-pom.xml

```diff
@@ -37,12 +37,6 @@
             <url>http://www.gnu.org/software/classpath/license.html</url>
             <distribution>repo</distribution>
         </license>
-
-        <license>
```

Comment:
The qualifiers in this package are under MIT license, aren't they?

Reply:
Yes, but the given license is the most restrictive one, and therefore the most useful one to state.
When a POM file contains two licenses, that generally means that the user is permitted to use either of the given licenses, at the user's option.  That is not the case for this project, and therefore giving two licenses was misleading.

Reply:
Ah, yes, that makes sense. Thanks for clarifying. Maybe a comment here would prevent confusion.